.sanity-ansible-ignore-2.12.txt

```diff
@@ -0,0 +1,42 @@
+plugins/modules/blivet.py import-2.6!skip
+plugins/modules/blivet.py import-2.7!skip
+plugins/modules/blivet.py import-3.5!skip
+plugins/modules/blivet.py import-3.6!skip
+plugins/modules/blivet.py import-3.7!skip
+plugins/modules/blivet.py import-3.8!skip
+plugins/modules/blivet.py import-3.9!skip
+plugins/modules/blivet.py import-3.10!skip
+plugins/modules/blivet.py validate-modules:import-error
+plugins/modules/blivet.py validate-modules:missing-gplv3-license
+plugins/modules/blivet.py validate-modules:module-invalid-version-added
+plugins/modules/blockdev_info.py import-2.7!skip
+plugins/modules/blockdev_info.py import-3.5!skip
```

Comment:
May I ask why blockdev_info.py, find_unused_disk.py, etc. are ok to skip just 2.7 and 3.5 (unlike blivet.py has to do all versions)?

Reply:
> May I ask why blockdev_info.py, find_unused_disk.py, etc. are ok to skip just 2.7 and 3.5 (unlike blivet.py has to do all versions)?

Good question.  I don't know why we don't get import errors with those files on other versions of python.